infra/gcp/clusters/projects/k8s-infra-public-pii/main.tf

```diff
@@ -0,0 +1,118 @@
+
+
+/**
+ * Copyright 2021 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id  = "k8s-infra-public-pii"
+  bucket_name = "k8s-infra-artifacts-gcslogs"
```

Comment:
Naming is really hard. I tempted to switch to `k8s-infra-gcr-audit-logs`. Not sure if it's for all the GCR registries or just scoped to `k8s-artifacts-prod`. 
@spiffxp WDYT ?

Reply:
tl;dr I'm happy with `k8s-infra-artifacts-gcslogs`.  

>Not sure if it's for all the GCR registries

GCR doesn't generate any access logs, and there are non-GCR buckets in k8s-artifacts-prod, e.g. https://github.com/kubernetes/k8s.io/blob/main/audit/projects/k8s-artifacts-prod/buckets/k8s-artifacts-kind/logging.json.  

> or just scoped to k8s-artifacts-prod.

Thinking with a mindset of bucket-level IAM vs. object-level ACLs, I wouldn't anticipate putting anything more than access logs related to artifacts here.  We might find we want to do something similar for `k8s-staging` buckets, so I'm fine not constraining to `artifacts-prod` in the name, but staying more general with `artifacts`